sound/soc/sof/sof-pci-dev.c

```diff
@@ -437,7 +437,7 @@ static const struct pci_device_id sof_pci_ids[] = {
 		.driver_data = (unsigned long)&tgl_desc},
 #endif
 #if IS_ENABLED(CONFIG_SND_SOC_SOF_ELKHARTLAKE)
-	{ PCI_DEVICE(0x8086, 0x4b55),
+	{ PCI_DEVICE(0x8086, 0x4b58),
```

Comment:
Is 0x4b55 a wrong number? I am not sure but should we add 0x4b58 and don't remove 0x4b55?

Reply:
yes, don't do this. Add the new PCI ID, never remove previous ones.

Reply:
OK

Reply:
And this should be in a separate patch, right?

Reply:
OK, I will make it separated.